.github/ISSUE_TEMPLATE/story.md

```diff
@@ -1,5 +1,5 @@
 ---
-name: 🎟  Story
+name: 🎟 Story
```

Comment:
I think the nbsp is deliberate, to get them to line up or something  (guessing this was a text editor onsave thing)